# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/rene/commonc++/gcc34.patch
# ROCK Linux is Copyright (C) 1998 - 2005 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---


The usual gcc-3.4 fixes.

  - Rene Rebe <rene@exactcode.de>

diff -ur commoncpp2-1.0.13-orig/include/cc++/thread.h commoncpp2-1.0.13/include/cc++/thread.h
--- commoncpp2-1.0.13-orig/include/cc++/thread.h	2003-06-07 01:16:45.000000000 +0200
+++ commoncpp2-1.0.13/include/cc++/thread.h	2004-05-04 18:03:06.000000000 +0200
@@ -1080,7 +1080,7 @@
 	static Thread* _main;
 
 	Thread *_parent;
-	enum Cancel _cancel;
+	Cancel _cancel;
 	Semaphore *_start;
 
 	// private data
